### PR TITLE
Implement gelu

### LINKED ISSUE #72: implement gelu
https://www.w3.org/TR/webnn/#api-mlgraphbuilder-gelu-method

### PR DESCRIPTION
**Pull request by BruceDai**
Fixed #72.

@fdwr @huningxin @mingmingtasd PTAL, thanks.

### CODE REVIEW

**Review by huningxin — APPROVED** [review]
LGTM!

**Review by fdwr — APPROVED** [review]
👍

File: src/unary.js [hunk 1]
```diff
@@ -18,19 +18,7 @@ export function unary(input, unaryFunc) {
...
-export const floor = (input) => unary(input, Math.floor);
-export const log = (input) => unary(input, Math.log);
-export const neg = (input) => unary(input, (x) => -1 * x);
-export const sin = (input) => unary(input, Math.sin);
-export const tan = (input) => unary(input, Math.tan);
-export const copy = (input) => unary(input, (x) => x);
-export const reciprocal = (input) => unary(input, (x) => 1 / x);
-export const sqrt = (input) => unary(input, Math.sqrt);
-export const erf = (input) => unary(input, (x) => {
+export function erfKernal(x) {
```
[the remark this answers is not in the document]
Comment (BruceDai) [reply src/unary.js]: Done in new commit, thanks.

**Review by fdwr — APPROVED** [review]
👍

**Pull request merged**